src/constructors/test/injectGlobal.test.js

```diff
@@ -19,6 +23,7 @@ describe('injectGlobal', () => {
 
   it(`should inject rules into the head`, () => {
     injectGlobal`
+      ${rule4}
```

Comment:
I doubt testing this in styled-components is necessary, mind removing this change?